ledger-core/virtual/integration/delegated_finished_test.go

```diff
@@ -1,65 +0,0 @@
-// Copyright 2020 Insolar Network Ltd.
-// All rights reserved.
-// This material is licensed under the Insolar License version 1.0,
-// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
-
-package integration
-
-import (
-	"context"
-	"testing"
-
-	"github.com/stretchr/testify/require"
-
-	"github.com/insolar/assured-ledger/ledger-core/application/builtin/proxy/testwallet"
-	"github.com/insolar/assured-ledger/ledger-core/insolar"
-	"github.com/insolar/assured-ledger/ledger-core/reference"
-	"github.com/insolar/assured-ledger/ledger-core/runner/call"
-	"github.com/insolar/assured-ledger/ledger-core/runner/executor/common/foundation"
-	"github.com/insolar/assured-ledger/ledger-core/runner/machine"
-	"github.com/insolar/assured-ledger/ledger-core/testutils/gen"
-	"github.com/insolar/assured-ledger/ledger-core/virtual/descriptor"
-	"github.com/insolar/assured-ledger/ledger-core/virtual/integration/utils"
-)
-
-func makeEmptyResult() []byte {
-	var ret0 error
-
-	return insolar.MustSerialize(foundation.Result{
-		Error:   nil,
-		Returns: []interface{}{ret0},
-	})
-}
-
-type callMethodFunc = func(ctx context.Context, callContext *call.LogicContext, code reference.Global, data []byte, method string, args []byte) (newObjectState []byte, methodResults []byte, err error)
-type callConstructorFunc = func(ctx context.Context, callContext *call.LogicContext, code reference.Global, name string, args []byte) (objectState []byte, result []byte, err error)
-
-func mockExecutor(t *testing.T, server *utils.Server, callMethod callMethodFunc, constructorFunc callConstructorFunc) {
```

Comment:
Why did you delete this file? By mistake?

Reply:
It was unused